python/tskit/trees.py

```diff
@@ -4809,14 +4826,49 @@ def write_vcf(
             pre 0.2.0 legacy behaviour of rounding values to the nearest integer
             (starting from 1) and avoiding the output of identical positions
             by incrementing is used.
-        """
+        :param nucleotide_alleles: If True, remaps existing alleles to
```

Comment:
I'm not super keen on this ``nucleotide_alleles`` option. I can see it having unexpected consequences down the line...

Reply:
I think we really want to make it possible for people to output legal VCFs, though. I'm not sure how else we'd do that.